### ISSUE TITLE
[Feature request] Use of custom garbage collector (or other jvm args)

### ISSUE THREAD

**Issue by Joly0** [FTBTeam/FTB-App#684]
**Is your feature request related to a problem? Please describe.**

I am trying to test out the shenandoah gc with various settings, though i am unable to change the default gc as it seems its hardcoded into the app here https://github.com/FTBTeam/FTB-App/blob/7b887cbdda58d647e9d3c4f1c5703315b6d4bff2/subprocess/src/main/java/net/creeperhost/creeperlauncher/Constants.java

**Describe the solution you'd like**

I would like to be able to change the default options and test out gcs other then g1gc and aswell be able to overwrite all the other default jvm args. 

A solution i have thought about would be removing the hardcoded default args and adding them to the settings.json in the .ftba\bin folder.
Another solution would be, to just ignore the default args if custom ones are set per instance.

The first one would have the advantage of being able to adjust multiple instances at once, the later would help with problems each individual instance might have with new args.

**Describe alternatives you've considered**

**Additional context**

**Comment by Joly0** [comment]
Still an issue

**Comment by Joly0** [comment]
??
This is still an issue

**Comment by Joly0** [comment]
@MichaelHillcox Can this please get reopened?

**Comment by Joly0** [comment]
The issue is still closed

**Comment by MinetaS** [comment]
@MichaelHillcox I believe this issue has been fixed by 46c28587f906acd022a50cbd5b1d7a67ab453402.

**Comment by MichaelHillcox** [comment]
Yup! This has been resolved now, you can adjust the default starting args of the JVM and the game to your hearts content :D
